### ISSUE TITLE
model training

### ISSUE THREAD

**Issue by fatiherbag** [OValery16/Tutorial-about-3D-convolutional-network#7]
I have 8 labels in my dataset. I get mismatch error when i try training.

size mismatch for module.fc7.weight: copying a param of torch.Size([8, 256]) from checkpoint, where the shape is torch.Size([27, 256]) in current model.

**Comment by OValery16** [comment]
do not include this layer in the module you restore via model.load_state_dict(...)
